libethereum/Executive.cpp

```diff
@@ -180,6 +180,11 @@ void Executive::initialize(Transaction const& _transaction)
 {
 	m_t = _transaction;
 
+	BlockHeader bh;
```

Comment:
@yann300 It makes sense for example for unconfirmed transactions coming over network to validate them before executing